constants/index.js

```diff
@@ -23,13 +23,21 @@ const ANALYTICS_EVENTS = Object.freeze({
 	CheckoutBegin: 'zerocodenft_checkout_begin',
 	CheckoutComplete: 'zerocodenft_checkout_complete',
 	CheckoutError: 'zerocodenft_checkout_failed',
-	ZeroCodeLinkClicked: 'zerocodenft_link_clicked'
+	ZeroCodeLinkClicked: 'zerocodenft_link_clicked',
 })
 
-
+const OS_SUPPORTED_CHAINS = Object.freeze({
```

Comment:
Chain IDs are not OS specific, they should be neutral, this way they can be reused in other logic if necessary. Let's rename this Enum to CHAIN_IDS

Reply:
Also, let's call chains as they are actually called for example EthereumMainnet = Ethereum, EthereumTestnet = Goerli, PolygonTestnet = Mumbai, etc. since just saying EthereumTestnet is too generic, there are more than 1 chains that are test, but ID = 5 refers specifically to Goerli.